packages/flutter/test_fixes/widgets.dart

```diff
@@ -177,4 +177,8 @@ void main() {
   final OverscrollIndicatorNotification notification = OverscrollIndicatorNotification(leading: true);
   final OverscrollIndicatorNotification notification = OverscrollIndicatorNotification(error: '');
   notification.disallowGlow();
+
+  // Changes made in
+  RawScrollbar rawScrollbar = RawScrollbar(isAlwaysShown: true);
+  nowShowing = rawScrollbar.isAlwaysShown;
```

Comment:
`nowShowing`  not defined?

Reply:
Ah thanks! This file is opted-out of analysis because it is meant to be broken code. 😆